Detectors/TPC/spacecharge/include/TPCSpaceCharge/SpaceCharge.h

```diff
@@ -910,93 +910,56 @@ class SpaceCharge
-    {mGlobalDistdR[Side::A], mGlobalDistdZ[Side::A], mGlobalDistdRPhi[Side::A], mGrid3D[Side::A], Side::A},
-    {mGlobalDistdR[Side::C], mGlobalDistdZ[Side::C], mGlobalDistdRPhi[Side::C], mGrid3D[Side::C], Side::C}}; ///< interpolator for the global distortions
-
-  DistCorrInterpolator<DataT> mInterpolatorLocalDist[FNSIDES]{
-    {mLocalDistdR[Side::A], mLocalDistdZ[Side::A], mLocalDistdRPhi[Side::A], mGrid3D[Side::A], Side::A},
-    {mLocalDistdR[Side::C], mLocalDistdZ[Side::C], mLocalDistdRPhi[Side::C], mGrid3D[Side::C], Side::C}}; ///< interpolator for the local distortions
-
-  DistCorrInterpolator<DataT> mInterpolatorLocalVecDist[FNSIDES]{
-    {mLocalVecDistdR[Side::A], mLocalVecDistdZ[Side::A], mLocalVecDistdRPhi[Side::A], mGrid3D[Side::A], Side::A},
-    {mLocalVecDistdR[Side::C], mLocalVecDistdZ[Side::C], mLocalVecDistdRPhi[Side::C], mGrid3D[Side::C], Side::C}}; ///< interpolator for the local distortion vectors
-
-  NumericalFields<DataT> mInterpolatorEField[FNSIDES]{
-    {mElectricFieldEr[Side::A], mElectricFieldEz[Side::A], mElectricFieldEphi[Side::A], mGrid3D[Side::A], Side::A},
-    {mElectricFieldEr[Side::C], mElectricFieldEz[Side::C], mElectricFieldEphi[Side::C], mGrid3D[Side::C], Side::C}}; ///< interpolator for the electric fields
+  inline static auto& mParamGrid = ParameterSpaceCharge::Instance();                                      ///<! parameters of the grid on which the calculations are performed
+  inline static int sNThreads{getOMPMaxThreads()};                                                        ///<! number of threads which are used during the calculations
+  inline static IntegrationStrategy sNumericalIntegrationStrategy{IntegrationStrategy::SimpsonIterative}; ///<! numerical integration strategy of integration of the E-Field: 0: trapezoidal, 1: Simpson, 2: Root (only for analytical formula case)
+  inline static int sSimpsonNIteratives{3};                                                               ///<! number of iterations which are performed in the iterative simpson calculation of distortions/corrections
+  inline static int sSteps{1};                                                                            ///<! during the calculation of the corrections/distortions it is assumed that the electron drifts on a line from deltaZ = z0 -> z1. The value sets the deltaZ width: 1: deltaZ=zBin/1, 5: deltaZ=zBin/5
+  inline static GlobalDistType sGlobalDistType{GlobalDistType::Fast};                                     ///<! setting for global distortions: 0: standard method,      1: interpolation of global corrections
+  inline static GlobalDistCorrMethod sGlobalDistCorrCalcMethod{GlobalDistCorrMethod::LocalDistCorr};      ///<! setting for  global distortions/corrections: 0: using electric field, 1: using local dis/corr interpolator
+  inline static SCDistortionType sSCDistortionType{SCDistortionType::SCDistortionsConstant};              ///<! Type of space-charge distortions
```

Comment:
if you want to forbid streaming, I think it should be `//!< ...`. But I'm not sure this is at all necessary for static variables.

For curiosity, why did you remove the comments below?

Reply:
Indeed, I think for static variables it doesnt matter, but `///<!` also works. see https://root.cern/manual/io/:
> To accommodate doxygen-style documentation, this annotation can also be written as ///<!.

> For curiosity, why did you remove the comments below?

I didnt remove them, they are just at the far right now, since it didnt work to exclude them from being streamed when they have a line break like 
```
  TriCubic mInterpolatorPotential[FNSIDES]{
    {mPotential[Side::A], mGrid3D[Side::A]},
    {mPotential[Side::C], mGrid3D[Side::C]}}; ///< interpolator for the potenial
```
so it is now `  TriCubic mInterpolatorPotential[FNSIDES]{{mPotential[Side::A], mGrid3D[Side::A]},{mPotential[Side::C],mGrid3D[Side::C]}};                                                                                                                                                                                                                                                                                                                                                                           ///<! interpolator for the potenial`